addons/website/static/src/builder/plugins/options/portal_option_plugin.js

```diff
@@ -0,0 +1,105 @@
+import { PortalOption, PortalCardVisibilityOption } from "./portal_option";
+import { Plugin } from "@html_editor/plugin";
+import { registry } from "@web/core/registry";
+import { withSequence } from "@html_editor/utils/resource";
+import { StyleAction } from "@html_builder/core/core_builder_action_plugin";
+import { BuilderAction } from "@html_builder/core/builder_action";
+
+class PortalOptionPlugin extends Plugin {
+    static id = "portalOption";
+    static dependencies = ["customizeWebsite"];
+
+    resources = {
+        builder_options: [
+            withSequence(1, PortalOption),
+            withSequence(2, PortalCardVisibilityOption),
+        ],
+        builder_actions: {
+            SetStylePortalCardAction,
+            CardVisibilityOptionAction,
+        },
+        clean_for_save_handlers: this.cleanForSave.bind(this),
+        immutable_link_selectors: [".o_portal_index_card > *"],
+        content_not_editable_selectors: [".o_portal_index_card [data-placeholder_count]"],
```

Comment:
```suggestion
        content_not_editable_selectors: [".o_portal_index_card:has(.alert)]"],
```
I prefer this one, if possible.